middlewares/cors.go

```diff
@@ -0,0 +1,100 @@
+package middlewares
+
+import (
+	"github.com/savsgio/atreugo/v10"
+
+	"errors"
+	"strconv"
+	"strings"
+)
+
+// Create new properties for CORS.
+type CorsOptions struct {
+	// given origin or origins can be shared with response
+	AllowedOrigins []string
+
+	// which headers can be used during the actual request
+	AllowedHeaders []string
+
+	// specify the method or methods allowed to resource
+	AllowedMethods []string
+
+	// which header names can be exposed as part of the response
+	ExposedHeaders []string
+
+	// whether a cached response can be used
+	AllowedVary []string
```

Comment:
Remove it

Reply:
OK, It will be removed after confirming my doubts, thank you.